src/systems/acoustic_comms/AcousticComms.cc

```diff
@@ -90,6 +158,19 @@ void AcousticComms::Load(
       _sdf->Get<double>("collision_time_per_byte");
   }
 
+  if (_sdf->HasElement("propagation_model"))
+  {
+    this->dataPtr->usePropagationModel = true;
+    sdf::ElementPtr propElement = _sdf->Clone()->
+                                   GetElement("propagation_model");
```

Comment:
Open to suggestions on how cloning can be avoided here. ``GetElement`` cannot be applied to const ``Element`` members it seems, and ``_sdf->GetElement("foo")`` throws an error that " `this` argument discards qualifiers."